X10D/src/Collections/BoolListExtensions.cs

```diff
@@ -26,7 +26,7 @@ public static byte PackByte(this IReadOnlyList<bool> source)
             throw new ArgumentNullException(nameof(source));
         }
 #endif
-
+        
```

Comment:
Can you remove this whitespace on line 29? The Source Validator job will fail if it sees trailing whitespace like this.